crates/red_knot_python_semantic/resources/mdtest/call/union.md

```diff
@@ -179,12 +179,15 @@ static_assert(is_subtype_of(Literal["a", ""], Union[Not[Literal[""]], Literal["a
 static_assert(is_subtype_of(Not[Literal[""]], Union[Not[Literal[""]], Literal["a", ""]]))
 
 def _(
-    x: Union[Literal["a", ""], Not[AlwaysFalsy]],
-    y: Union[Literal["a", ""], Not[Literal[""]]],
+    a: Union[Literal["a", ""], Not[AlwaysFalsy]],
+    b: Union[Literal["a", ""], Not[Literal[""]]],
+    c: Union[Literal[""], Not[Literal[""]]],
+    d: Union[Not[Literal[""]], Literal[""]],
```

Comment:
nit: is there a reason why we're using the pre-PEP-604 syntax here...? I find it more readable if we can keep the bracketese to a minimum:

```suggestion
    a: Literal["a", ""] | Not[AlwaysFalsy],
    b: Literal["a", ""] | Not[Literal[""]],
    c: Literal[""] | Not[Literal[""]],
    d: Not[Literal[""]] | Literal[""],
```

Reply:
Once you start bracketing, it's just hard to stop.